doc/src/reference/py.rst

```diff
@@ -121,7 +121,10 @@ target
 
     Constructs the target.
 
-    :param str name: The name of the target to construct. This can either be 'cpu' or 'gpu'.
+    :param str name: The name of the target to construct. MIGraphx supports `cpu`, `gpu` and `ref` as targets. `ref` provides referecnce 
```

Comment:
I actually dont think we should try to iterate all the targets here. We should probably have a different page to describe all of our targets.

Reply:
For a quick fix, just list `gpu` and `ref`, and then we can open an issue to to add a page for backend targets.

Reply:
Opend #1154